### PR TITLE
Minor mutations-iterator fixes

### PR DESCRIPTION
**Pull request by SR4ven**
While looking at the changes, I came across some inconsistencies.

The tests will fail because there seems to be a problem with absolute name resolving that I described in https://github.com/jtpereyda/boofuzz/pull/455#issuecomment-670221341.

### CODE REVIEW

**Comment by SR4ven** [comment]
Waiting for #458 to be merged

**Comment by jtpereyda** [comment]
Pending discussion in #458.

**Comment by SR4ven** [comment]
A part of this PR switches to the new name resolving so we consistently use it everywhere.
Some tests ware failing before #458 was merged, that's why I left the message above.

However, right now we should be independent of #458. We can merge this one and if we change the name resolving, we fix the tests at the same time.

**Review by jtpereyda — APPROVED** [review]

**Pull request merged**
